superset/errors.py

```diff
@@ -138,10 +139,12 @@ class SupersetErrorType(str, Enum):
     1034: _("The port number is invalid."),
     1035: _("Failed to start remote query on a worker."),
     1036: _("The database was deleted."),
+    1037: _("Custom SQL does not allow subquery."),
```

Comment:
```suggestion
    1037: _("Custom SQL fields cannot contain subqueries."),
```

(keeping it consistent with Aaron's suggestion)